src/sonic-config-engine/minigraph.py

```diff
@@ -124,6 +124,7 @@ def parse_dpg(dpg, hname):
 
         ipintfs = child.find(str(QName(ns, "IPInterfaces")))
         intfs = []
+        intfnames = {}
```

Comment:
I think so. in order to get all the interface names, we could use this. the list returned will have duplicates.